internal/cli/get.go

```diff
@@ -321,3 +399,33 @@ func (o *GetOptions) processResponse(response interface{}, err error, kind strin
 	// For table format, pass the full response
 	return formatter.Format(response, options)
 }
+
+// getListMetadata extracts the ListMeta and the number of items from a list response.
+func getListMetadata(response interface{}) (*api.ListMeta, int, error) {
+	json200, err := responseField[interface{}](response, "JSON200")
+	if err != nil {
+		return nil, 0, err
+	}
+
+	// Retrieve metadata
+	listMeta, err := responseField[api.ListMeta](json200, "Metadata")
+	if err != nil {
+		return nil, 0, err
+	}
+
+	// Determine number of items via reflection
+	v := reflect.ValueOf(json200)
```

Comment:
nit: whenever I see the use of `reflect I` wonder is this really needed? Is there no other way to get the data we need from the response? If we have explored that and using reflection is the only way then that is fine. But please explore. high-level reflection is very expensive in this context it might not be a big deal but its good practice to try and avoid.